sdks/python/Makefile

```diff
@@ -12,7 +12,7 @@ WD                    := $(shell echo "`pwd`/client")
 DOCKER = docker run --rm -v $(WD):/wd --workdir /wd
 CHOWN = chown -R $(shell id -u):$(shell id -g)
 
-publish: generate
+publish:
```

Comment:
This was there for testing in the CI: https://github.com/argoproj/argo-workflows/blob/master/.github/workflows/sdks.yaml#L20